src/util/transition/ngbTransition.ts

```diff
@@ -47,18 +48,15 @@ export const ngbRunTransition =
             }
           }
 
-          // If animations are disabled, we have to emit a value and complete the observable
-          if (!options.animation) {
-            (startFn(element, context) || noopFn)();
-            return of(undefined);
-          }
+          reflow(element);
```

Comment:
Could you please explain why in a comment? because we'll never remember this later.